_topic_maps/_topic_map.yml

```diff
@@ -3230,38 +3230,50 @@ Topics:
     File: cnf-image-based-upgrade-base
   - Name: Performing an image-based upgrade for single-node OpenShift clusters using GitOps ZTP
     File: ztp-image-based-upgrade
-# Remove RDS from published docs until 4.16 RDS is released
-# ---
-# Name: Reference design specifications
-# Dir: telco_ref_design_specs
-# Distros: openshift-telco
-# Topics:
-# - Name: Telco reference design specifications
-#   File: telco-ref-design-specs-overview
-# - Name: Telco RAN DU reference design specification
-#   Dir: ran
-#   Topics:
-#   - Name: Telco RAN DU reference design overview
-#     File: telco-ran-ref-design-spec
-#   - Name: Telco RAN DU use model overview
-#     File: telco-ran-du-overview
-#   - Name: RAN DU reference design components
-#     File: telco-ran-ref-du-components
-#   - Name: RAN DU reference design configuration CRs
-#     File: telco-ran-ref-du-crs
-#   - Name: Telco RAN DU software specifications
-#     File: telco-ran-ref-software-artifacts
-# - Name: Telco core reference design specification
-#   Dir: core
-#   Topics:
-#   - Name: Telco core reference design overview
-#     File: telco-core-rds-overview
-#   - Name: Telco core use model overview
-#     File: telco-core-rds-use-cases
-#   - Name: Core reference design components
-#     File: telco-core-ref-design-components
-#   - Name: Core reference design configuration CRs
-#     File: telco-core-ref-crs
+- Name: Image-based installation for single-node OpenShift clusters
+  Dir: image-base-install
+  Topics:
+  - Name: Understanding an image-based installation for single-node OpenShift clusters
+    File: ibi-understanding-image-based-install
+  - Name: Preparing for an image-based installation for single-node OpenShift clusters
+    File: ibi-preparing-for-image-based-install
+  - Name: Performing an image-based installation for single-node OpenShift clusters
+    File: ibi-factory-image-based-install
+  - Name: Performing an image-based deployment for managed single-node OpenShift clusters
+    File: ibi-edge-image-based-install
+  - Name: Performing an image-based deployment for standalone single-node OpenShift clusters
+    File: ibi-edge-image-based-install-standalone
+---
+Name: Reference design specifications
```

Comment:
Why are you adding these here?

Reply:
Whoops, dunno. Probably copy-paste error. Thanks!